README.md

```diff
@@ -6,8 +6,27 @@ We will be building an API for the purpose of accessing application data program
 
 Your database will be PSQL, and you will interact with it using [node-postgres](https://node-postgres.com/).
 
+## Setup
+
+### Connect to the databases locally
+
 You will need to create two .env files for your project: .env.test and .env.development. Into each, add PGDATABASE=<database_name_here>, with the correct database name for that environment (see /db/setup.sql for the database names). Double check that these .env files are .gitignored.
 
+## 1. GET /api/topics
```

Comment:
You can get rid of lines 13 to 32 in the readme as these won't be in your final readme :)